spec/time_field_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe Timely::TimeField do
+  let(:klass) {
+    Class.new do
+      include Timely::TimeField
+
+      def initialize(attrs)
+        @start_time = attrs[:start_time]
+        @end = attrs[:end]
+      end
+
+      def read_attribute(attr)
+        instance_variable_get("@#{attr}".to_sym)
+      end
+
+      time_fields :start_time, :end
+    end
+  }
+
+  subject {
+    klass.new(
+      start_time: DateTime.new(2000, 1, 1, 11, 59, 0),
```

Comment:
Do not use `DateTime.new` without zone. Use one of `Time.zone.local`, `DateTime.new.current`, `DateTime.new.in_time_zone`, `DateTime.new.utc`, `DateTime.new.getlocal`, `DateTime.new.iso8601`, `DateTime.new.jisx0301`, `DateTime.new.rfc3339`, `DateTime.new.to_i`, `DateTime.new.to_f` instead.